### ISSUE TITLE
PySide6 Linking

### ISSUE THREAD

**Issue by psydox** [Nuitka/Nuitka#2902]
I would like to inquire regarding how Nuitka compiles a Python script with PySide6.  

This is something related to licensing.  Is it ok to use the LGPL side of PySide6 when the Python script is compiled with Nuitka?  Or does Nuitka statically compile (some parts) of Pyside6 with the Python scripts?

Kind regards.

**Comment by kayhayen** [comment]
Legal questions are best answered by lawers.

**Comment by psydox** [comment]
Hi 

> Legal questions are best answered by lawyers.

Hi kayhayen,

Thank you for the answer.

I already discussed it with our lawyer.  She mentioned that as long as there is no static linking of the PySide6 when Nuitka compiled the apps then there is no issue.

I did check the compiled version of my apps and there are those PySide6 .pyd included on the folder.

I asked this because I am not sure if Nuitka compiles (some part) of PySide6 and links it with the final binary on Windows.

Anyways, thanks!


Kind regards.
